extension/src/cli/constants.ts

```diff
@@ -1,4 +1,5 @@
 export const MIN_CLI_VERSION = '2.11.0'
+export const BUILD_CLI_VERSION = '2.12.0'
```

Comment:
[Q] Should this be `2.13.0`?
[C] Could name it `LATEST_TESTED_CLI_VERSION`.

Reply:
done!